elizhahn/news-box
# Styling/responsive breaks

## What is being added/changed?
- responsive styling was added to all current views

## How should this be tested?
- for each view use the dev tools to set width from 280 up to 1500, there should be no awkward break points or layout issues

## future iterations?
- Better styling for large monitors

## Linked Issues?
- #19
